components/organisms/AboutSection/about-section.styled.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components';
+
+import { FramedImage } from '@components/atoms';
+
+export const StyledAboutSection = styled.section`
+    display: grid;
+    grid-template-columns: 60% 40%;
+    grid-template-rows: 15% 45% 20% 20%;
+    gap: 3px 3px;
+    grid-template-areas:
+        'blank header'
+        'text image'
+        'more-link image'
+        'footer image';
```

Comment:
Look into other grid units you can use. `%` is one way, but `fr` can also be very useful, and while similar does offer more flexibility when you use different units.

Something that is also good to look into is the [css grid minmax function](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax())